src/main/java/org/codehaus/plexus/util/xml/PrettyPrintXMLWriter.java

```diff
@@ -247,6 +249,19 @@ private static String escapeXmlAttribute(String text) {
         return b.toString();
     }
 
+    private static String escapeXmlText(String text) {
+        text = escapeXml(text);
+
+        Matcher m = lowersText.matcher(text);
+        StringBuffer b = new StringBuffer();
```

Comment:
StringBuilder builder

Reply:
`appendReplacement` require `StringBuffer`, `StringBuilder` is supported from JDK 9